### ISSUE TITLE
htmlmeta package?

### ISSUE THREAD

**Issue by rpodcast** [gadenbuie/drake-intro#3]
This is a fantastic intro to `drake`!  I'd like to run the slides on my machine but I couldn't find the `htmlmeta` package on CRAN or GitHub.  Could you point me to where I can find it?

**Comment by gadenbuie** [comment]
Hey @rpodcast, I need to fix that, thanks for reminding me. It's a package I've been working on that I'll make public soon and it was silly of me to include it here (and it already has a new name 🙄). In the mean time, try rolling back to 2dc74b1fe6fa01b2c17c56b14c9187a805c46aa7 -- none of the important content changed since then.

Also if you're compiling from the Rmd source, I used the dev branch of [xaringanthemer@dev](https://github.com/gadenbuie/xaringanthemer/tree/dev). **Edit:** I see now that I included the CSS output and you won't need xaringanthemer to compile.

Feel free to let me know here if there are any other sneaky dependencies I forgot about.
